README.md

```diff
@@ -0,0 +1,41 @@
+# GovernmentBillSummarizer
+
+Use NLP to classify and summarize bills available through the [govinfo API](https://api.govinfo.gov/docs/).
+
+## Using
+
+To run program correctly, the program must be initialized by running `UserInterface.py`:
+
+```bash
+python UserInterface.py
```

Comment:
Is this correct? 

Since the README is now in the top-level directory and `UserInterface.py` is in `Classifier` sub-directory, I could update this to:

```
python Classifier/UserInterface.py
```

so it's more copy and pastable.